src/utils/openmc/universe.py

```diff
@@ -911,8 +880,24 @@ def create_xml_subelement(self, xml_element):
         if not test is None:
             return
 
-        lattice_subelement = \
```

Comment:
I'd like to, but I couldn't see how to do it easily since the abstract method would need the `lattice_subelement` variable, and that variable needs to be created in a type-specific manner since one requires `<lattice>` and the other requires `<hex_lattice>`.  Maybe we should make `lattice_subelement` an attribute of the `Lattice` class so that it can be accessed in the abstract method?